.github/workflows/ci-scripts.yml

```
@@ -0,0 +1,47 @@
+name: CI Scripts
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  golangci:
+    if: ${{ github.event_name == 'pull_request' || github.event_name == 'schedule' }}
```

```suggestion
    if: ${{ github.event_name == 'pull_request' }}
```

Since there aren't any schedule triggers, wanna remove it from the conditional?